### PR TITLE
Avax C-Chain Script

### PR DESCRIPTION
**Pull request by cyje66**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,8 @@
 **/.nb-gradle/
 **/node_modules/
 /bin/
+/.idea
+script-generator/coolwallet-signing-sdk.iml
+script-generator/target
+script-generator/${project.build.directory}
+script-generator/local-lib
```
Comment (hank121314) [inline .gitignore:15]: Unrelated changes.
The `.idea` should be in your global gitignore.

File: script-generator/pom.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
     </repositories>
     <dependencies>
         <!-- https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on -->
-        <!--        <dependency>
```
Comment (hank121314) [inline script-generator/pom.xml:17]: Unrelated changes.

**Review by hank121314 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -7,4 +7,4 @@
 **/nbdist/
 **/.nb-gradle/
 **/node_modules/
-/bin/
+/bin/
```
Comment (hank121314) [inline .gitignore:10]: Add new line EOF.

File: script-generator/pom.xml [hunk 4]
```diff
@@ -33,4 +33,4 @@
         <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
     <name>coolwallet-signing-sdk</name>
-</project>
+</project>
```
Comment (hank121314) [inline script-generator/pom.xml:36]: Add new line EOF.

**Review by hank121314 — CHANGES_REQUESTED** [review]
Please make sure your PR does not contain any changes except `.java` files.

File: .gitignore (hunk 3 above)
Comment (cyje66) [reply .gitignore:10]: ok!

**Review by hank121314 — APPROVED** [review]

**Review by hankliu418 — APPROVED** [review]

**Pull request merged**